topdown/http.go

```diff
@@ -121,6 +129,13 @@ func executeHTTPRequest(bctx BuiltinContext, obj ast.Object) (ast.Value, error)
 				return nil, err
 			}
 			body = bytes.NewBuffer(bodyValBytes)
+		case "headers":
+			headersVal := obj.Get(val).Value
+			headersValInterface, err := ast.JSON(headersVal)
+			if err != nil {
+				return nil, err
+			}
+			customHeaders = headersValInterface.(map[string]interface{})
```

Comment:
This will panic if someone passes a non-object value. Change this line to:

```
var ok bool
customHeaders, ok = headersValInterface.(map[string]interface{})
if !ok {
  return nil, fmt.Errorf("invalid type for headers key")
}
```

Reply:
Took care of this one